crates/observe/src/config.rs

```diff
@@ -0,0 +1,79 @@
+use tracing::level_filters::LevelFilter;
+
+#[derive(Debug, Clone)]
+pub struct ObserveConfig {
+    /// https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html
+    pub env_filter: String,
+    /// Minimum level threshold for stderr output
+    pub stderr_threshold: LevelFilter,
+    /// Output log events as JSON
+    pub use_json_format: bool,
+    /// Tracing config
+    pub tracing: Option<TracingConfig>,
+}
+
+impl ObserveConfig {
+    /// Create a new TracingConfig
```

Comment:
nit: should be `ObserveConfig` in the comment, although I would even add the comment line in this case as the code is quite clear